hera_sim/defaults.py

```diff
@@ -51,17 +51,19 @@ class _Defaults:
     hera_sim.defaults(module, model)
     """
 
-    def __init__(self, config_file='h1c'):
-        self._config = self._get_config(config_file)
+    def __init__(self, config='h1c'):
+        self._config = self._get_config(config)
         self._check_config()
         self._use_season_defaults = False
     """Load in a configuration and check its formatting.
 
     Parameters
     ----------
-    config_file : str, optional (default 'h1c') 
-        May either be an absolute path to a configuration YAML or one of
-        the observing season keywords: {}
+    config : str or dict, optional (default 'h1c') 
+        May either be an absolute path to a configuration YAML, one of
+        the observing season keywords ({}), or a dictionary with the
```

Comment:
not sure what the `({})` is doing here?

Reply:
I had intended to use string formatting to fill that in, but I realize that I didn't ever actually format the string. I'll just fill in the observing season keywords manually.